app/models/topic_tracking_state.rb

```diff
@@ -40,6 +40,10 @@ class TopicTrackingState
                 :notification_level,
                 :tags
 
+  def self.channel
```

Comment:
Why make this a self method instead of a const?

Reply:
Good point I've made the change.